test/Generator/ParameterGeneratorTest.php

```diff
@@ -118,6 +118,7 @@ public function testFromReflectionGetDefaultValue()
 
     /**
      * @group 95
+     * @requires PHP < 8.1
```

Comment:
I'm skipping the test on PHP 8.1 because PHP has changed its behavior here.

See https://3v4l.org/BocvA/rfc#vgit.master Vs. https://3v4l.org/BocvA